e2e/config.json

```diff
@@ -0,0 +1,4 @@
+{
+    "setupTestFrameworkScriptFile": "./init.js",
```

Comment:
Please use two-space indentation instead of four-space per the general rule of thumb with JSON in this repo.

Reply:
Fixed.